server/apps/acсount/models.py

```diff
@@ -0,0 +1,200 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.db.models import TextChoices, QuerySet
+from PIL import Image
+from django.core.exceptions import ValidationError
+from django.core.validators import FileExtensionValidator
+from typing import Any
+from django.utils import timezone
+    
+
+def validator_avatar_weight(file: Any) -> None:
+    """Валидатор размера файла для изображений."""
+    limit = 5 * 1024 * 1024
+    if file.size > limit:
+        raise ValidationError('нюхай бебру')
+
+
+class User(AbstractUser):
+    class GENDERS(TextChoices):
+        MALE = 'male', 'Male'
+        FEMALE = 'female', 'Female'
+
+    gender = models.CharField(
+        max_length=6,
+        choices=GENDERS.choices,
+        default=GENDERS.MALE,
```

Comment:
Why is the user `male` by default???

It would be better if u allow the `None` value and set it to the default.